src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -179,6 +187,48 @@ export class HuiCardPicker extends LitElement {
         !UNAVAILABLE_STATES.includes(this.hass!.states[eid].state)
     );
 
+    this._loadCards();
+
+    this.requestUpdate();
+  }
+
+  private _loadCards() {
+    const cards: Card[] = previewCards
+      .map((type: string) => ({
+        type,
+        name: this.hass!.localize(`ui.panel.lovelace.editor.card.${type}.name`),
+        description: this.hass!.localize(
+          `ui.panel.lovelace.editor.card.${type}.description`
+        ),
+      }))
+      .concat(
+        nonPreviewCards.map((type: string) => ({
+          type,
+          name: this.hass!.localize(
+            `ui.panel.lovelace.editor.card.${type}.name`
+          ),
+          description: this.hass!.localize(
+            `ui.panel.lovelace.editor.card.${type}.description`
+          ),
+          noElement: true,
+        }))
+      );
+    if (customCards.length > 0) {
+      cards.concat(
+        customCards.map((ccard: CustomCardEntry) => ({
+          type: ccard.type,
+          name: ccard.name,
+          description: ccard.description,
+          noElement: true,
```

Comment:
Why don't we render custom cards?

Reply:
Oops missed that one

Reply:
actually it seems to be right: `if (!noElement || customCard?.preview) {`